src/props-invl.h

```diff
@@ -16,25 +16,25 @@
  *   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
  */
 
-
 #ifndef __PROPS_INVL_H__
 #define __PROPS_INVL_H__
 
 #include "proj.h"
 
 typedef struct EditIntervalDialog_s EditIntervalDialog;
 
-EditIntervalDialog * edit_interval_dialog_new (void);
-void edit_interval_dialog_destroy(EditIntervalDialog *dlg);
+EditIntervalDialog *
+edit_interval_dialog_new (void);
+void
+edit_interval_dialog_destroy (EditIntervalDialog *dlg);
 
-void edit_interval_set_interval (EditIntervalDialog *dlg, GttInterval *ivl);
+void
+edit_interval_set_interval (EditIntervalDialog *dlg, GttInterval *ivl);
 
 /* pop up a dialog box for editing an interval */
-void edit_interval_dialog_show(EditIntervalDialog *dlg);
-void edit_interval_set_close_callback (EditIntervalDialog *dlg,
-									   GCallback f,
-									   gpointer data);
-
-
+void
+edit_interval_dialog_show (EditIntervalDialog *dlg);
+void
+edit_interval_set_close_callback (EditIntervalDialog *dlg, GCallback f, gpointer data);
```

Comment:
I usually have the return type and function name on the same line, but separate should be fine too.

Ideally it does mean there should always be an empty line after each definition, perhaps *especially* if there is no function comment block. I feel it floats together a bit when the empty lines are missing like in this case.

It could be dealt with on a case-by-case basis when coming across it.

Reply:
Having the return type on a separate line was new to me too, but quite some parts of this codebase have it like this. Unfortunately there does not seem to be a coherent pattern. I will change it back to have the return type and the name on the same line.